File-Formats/PDF/Convert-HTML-To-PDF/Windows-Forms.md

```diff
@@ -11,96 +11,94 @@ keywords: Assemblies
 
 The Syncfusion HTML to PDF converter is a .NET library used to convert HTML or web pages to PDF. Using this library can convert HTML to PDF in Windows Forms application.
```

Comment:
The Syncfusion HTML to PDF converter is a .NET library used to convert HTML or web pages to PDF document in Windows Forms application.

Reply:
content changed